### ISSUE TITLE
[Bug] REST call saveDevice modifies "createdTime" when changing label for an existing device.

### ISSUE THREAD

**Issue by chenybing** [thingsboard/thingsboard#6523]
**Describe the bug**
When using REST call saveDevice to modify the label of a device, the label is successfully changed but the createdTime is also changed into a date in 1969.

**Your Server Environment**
* own setup
  * local infrastructure
  * ThingsBoard 3.3.4.1

**Your Client Environment**
**Desktop (please complete the following information):**

* OS: Ubuntu 20.04
* Browser: Chrome

* Connectivity
  * HTTP

**To Reproduce**
1. Create a device manually on ThingsBoard as a Tenant user.  Copy the device ID, the device profile ID, the device access token, and the tenant user ID.
2. Go to the Swagger UI for ThingsBoard, go to device_profile > saveDevice
3. Click on "Try it out"
4. Input all the fields needed to change the label for the device (please see screenshots for what was put in the data).
5. Submit.

**Expected behavior**
Server returns 200 success, the label for the device is updated, but the createdTime reverted to a date in 1969.

**Screenshots**
1. Initial view of device with the correct createdTime timestamp.
![01](https://user-images.githubusercontent.com/2068386/166726725-bb4922c4-dafa-451e-83b9-1b3c205d5075.PNG)

2. The content of data being sent to saveDevice REST call.
![02](https://user-images.githubusercontent.com/2068386/166726749-32e10e8f-1b32-412b-b0b6-c2beeb12e5be.PNG)

3. The crafted message to be sent to ThingsBoard.
![03](https://user-images.githubusercontent.com/2068386/166726764-94409e8c-3c71-4db2-8357-e2918ea13985.PNG)

4. The success response from ThingsBoard after changing the label for an existing device.
![04](https://user-images.githubusercontent.com/2068386/166726783-f2c02001-5d04-4e82-b08d-64cf4afa7018.PNG)

5. The createdTime for the device is changed to 1969 which is incorrect.
![05](https://user-images.githubusercontent.com/2068386/166726803-6d048f0c-cf04-4696-af34-fd8e13087e0b.PNG)

**Comment by pbranchi** [comment]
I faced a similar issue:  setting only the "additional info -> gateway" field resulted in all other unspecified data being reset. 

From my understanding, all fields missing from the request body is reset/set to 0: think about it as a form empty field.

What I did as a workaround was:
1. get all the device the data from thingsboard, with 
![image](https://user-images.githubusercontent.com/6998161/167036685-07187414-1bae-4b62-8c8c-042efff92909.png)
2. modify the fields that needs to be modified
3. send the whole (modified) payload back

This way no data is lost/reset
